packages/pds/src/account-manager/oauth-store.ts

```diff
@@ -181,6 +181,13 @@ export class OAuthStore
           const account = await this.accountManager.getAccount(did)
           if (!account) throw new Error('Account not found')
 
+          if (account.email && !account.emailConfirmedAt) {
+            await this.accountManager.sendConfirmEmail({
+              did,
+              email: account.email,
+            })
+          }
+
```

Comment:
The rationale for adding this here is that accounts created via XRPC calls likely will want more fine-grained control over when the email confirmation is sent, where as via open registration, you probably want to always send confirmation.